include/seqan3/search/fm_index/detail/csa_alphabet_strategy.hpp

```diff
@@ -162,6 +162,30 @@ namespace sdsl
             m_C.load(in);
             read_member(m_sigma, in);
         }
+
+        template <typename archive_t>
+        void CEREAL_SAVE_FUNCTION_NAME(archive_t & ar) const
+        {
+            ar(CEREAL_NVP(m_C));
+            ar(CEREAL_NVP(m_sigma));
```

Comment:
doesn't NVP mean named-value-pair? Where is the name?

Reply:
the name is the variable name.
The macro ist short hand for, e.g., `cereal::make_nvp("m_C", m_C);`

Reply:
Yes, but I mean, do you need the names if they are as useless as they currently are? :wink:

Reply:
Expressive uselessness? 🤷‍♂️